.jenkins/validate_tutorials_built.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+from typing import List
+
+from bs4 import BeautifulSoup
+
+REPO_ROOT = Path(__file__).parent.parent
+
+# files that are ok to have 0 min 0 sec time, probably because they don't have any python code
+OK_TO_NOT_RUN = [
+    "beginner/basics/intro.html",
+]
+
+# when the tutorial is fixed, remove it from this list
+KNOWN_BAD = [
```

Comment:
Can we call it something other than `bad`?

Some tutorials are intentionally disabled. TTS tutorial is disabled because conflicting dependency issue but the original tutorial is executed on TorchAudio CI and we make sure it works.

Reply:
which tutorial is this? i can put it in the `ok_to_not_run` list, which isnt really a better name
the end goal is that we fix everything in the known bad list or decide that its ok to to run (b/c it has no code or some other reason)